website/landing/components/common/Install.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Button, Text } from ".";
+
+const DEFAULT_COMMAND = "npm install @codesandbox/sandpack-react";
```

Comment:
What do you think if you bring these "global" values into a single place (website.config.json?)? I'd add the project name there too.

Reply:
Adjusted here: https://github.com/codesandbox/sandpack/pull/129/commits/fe320823f2bef8f4c445488a6b896661df39004f.